MC/analysis_testing/o2dpg_analysis_test_workflow.py

```diff
@@ -202,12 +203,16 @@ def get_additional_workflows(input_aod):
                     found_O2zdc_001 = True
                 if "O2bc_001" in j.GetName():
                     found_O2bc_001 = True
+                if "O2trackextra_001" in j.GetName():
+                    found_O2trackextra_001 = True 
             if not found_O2collision_001:
                 additional_workflows.append("o2-analysis-collision-converter --doNotSwap")
             if not found_O2zdc_001:
                 additional_workflows.append("o2-analysis-zdc-converter")
             if not found_O2bc_001:
                 additional_workflows.append("o2-analysis-bc-converter")
+            if found_O2trackextra_001:
```

Comment:
And I guess now we need a `not` here as well, right?
Just to be sure :)

Reply:
.... of course..... thank you very much Benedikt, sorry for the mess

Reply:
no worries, thanks for the quick action!